File: commerce-order-engine-java/src/main/java/com/example/commerce/order/settlements/SettlementModels.java
package com.example.commerce.order.settlements;
import java.math.BigDecimal;import java.time.*;import java.util.List;
public final class SettlementModels{private SettlementModels(){}public record DailySettlement(LocalDate settlementDate,BigDecimal paymentAmount,BigDecimal refundAmount,BigDecimal netAmount,long paymentCount,long refundCount,Instant calculatedAt){}public record SettlementPage(List<DailySettlement> items,LocalDate from,LocalDate to){}public record RebuildResult(LocalDate from,LocalDate to,int rebuiltDays){} }

File: commerce-operations-dotnet/src/Commerce.Operations.Api/Settlements/SettlementModels.cs
namespace Commerce.Operations.Api.Settlements;
public sealed record DailySettlement(DateOnly SettlementDate,decimal PaymentAmount,decimal RefundAmount,decimal NetAmount,long PaymentCount,long RefundCount,DateTime CalculatedAt);
public sealed record SettlementPage(IReadOnlyList<DailySettlement> Items,DateOnly From,DateOnly To);
public sealed record RebuildResult(DateOnly From,DateOnly To,int RebuiltDays);
